libslide/src/partial_evaluator.rs

```diff
@@ -134,6 +138,8 @@ mod tests {
         unwrap_brackets_nested:           "[a] + [1]" => "a + 1"
 
         flattened_addition:             "1 + 2 - b + 3 - b" => "6 - b - b"
+
+        issue_92: "a + 1 - 1" => "a"
     }
```

Comment:
No. We will not be naming these things by issue number- especially rules. Call it literal_cancellation or something like that.

Reply:
Why not? These tests are mostly for regressions anyway. If we have a similar issue for something like `a + b + 9 + c -e -9` where 9 - 9 doesn't go to zero, what do we name this? `literal_cancellation_2` is fine but it leaves less context about when and where the bug appeared.

Related to this, I think we should move all tests into some kind of corpus (it can even just be a giant text document), similar to how rust/rustfmt do it. What do you think?